source/guides/module_guides/bgtm.md

```diff
@@ -58,256 +60,323 @@ In terms of class structure we recommend the following (more detail below):
-
-#### Containment and Anchoring
-
-Classes do not _automatically_ contain the classes they declare. (This is because classes can be declared in several places via `include` and similar functions. Most of these places shouldn't contain the class, and trying to contain it everywhere would cause huge problems.)
-
-Thus, if you want to allow other modules to form ordering relationships with your module, you should ensure that your main class(es) explicitly _contain_ any subordinate classes they declare. For more information and context, you can see [the Containment page of the Puppet reference manual](/puppet/latest/reference/lang_containment.html).
+```
 
-In Puppet 3.4.0 / Puppet Enterprise 3.2 and later, you can contain classes by using [the `contain` function](/puppet/latest/reference/function.html#contain) on them. To support versions prior to 3.4.0 / PE 3.2, you must use the **anchor pattern** to hold those classes in place. See below for an example of anchoring. (*Note: anchoring requires [puppetlabs-stdlib](http://forge.puppetlabs.com/puppetlabs/stdlib)*.)
+Rather than making a `require` to several packages, the above ordering allows you to refactor and improve `module::install` without adjusting the manifests of other classes to match the changes.
 
-Two resources to anchor things to:
+#### Containment and anchoring
 
-~~~ ruby
-    anchor { 'module::begin': }
-    anchor { 'module::end' }
-~~~
+To allow other modules to form ordering relationships with your module, ensure that your main classes explicitly _contain_ any subordinate classes they declare.
 
-The anchoring:
+Classes do not _automatically_ contain the classes they declare. This is because classes can be declared in several places via `include` and similar functions. To contain classes, use [the `contain` function](/puppet/latest/function.html#contain). For more information and context about containment, see [the containment docs](/puppet/latest/lang_containment.html).
 
-~~~ ruby
-    Anchor['module::begin'] ->
-      Class['module::install'] ->
-      Class['module::config']  ->
-      Class['module::service'] ->
-    Anchor['module::end']
-~~~
+For example, the `ntp` module uses containment in the main `ntp` class:
 
-This enforces ordering, so `::install` will happen before `::config`, and
-`::config` before `::service`.
+```
+contain ntp::install
+  contain ntp::config
+  contain ntp::service
+  Class['::ntp::install'] ->
+  Class['::ntp::config'] ~>
+  Class['::ntp::service']
+```
 
-For a real world example of anchoring in action, please see Hunter's [puppet-wordpress](https://github.com/hunner/puppet-wordpress/blob/master/manifests/init.pp) module.
+Containment is supported in Puppet 3.4 and later. To support versions prior to Puppet 3.4 (or Puppet Enterprise 3.2), you must use the [anchor pattern][anchor] to hold those classes in place. Anchoring requires [puppetlabs-stdlib](http://forge.puppetlabs.com/puppetlabs/stdlib).
 
```

Comment:
It seems like we shouldn't even mention anchoring because Puppet 3 is now EOL and if we're not going to include an example.